nlptest/modelhandler/spacy_modelhandler.py

```diff
@@ -149,3 +149,60 @@ def predict_raw(self, text: str) -> List[str]:
     def __call__(self, text: str, return_all_scores: bool = False, *args, **kwargs) -> SequenceClassificationOutput:
         """Alias of the 'predict' method"""
         return self.predict(text=text, return_all_scores=return_all_scores, **kwargs)
+
+class PretrainedModelForTranslation(_ModelHandler):
+    """
+    Args:
+        model: Pretrained SpaCy pipeline.
+    """
+
+    def __init__(
+            self,
+            model
+    ):
+        annotation = getattr(model, '__call__').__annotations__
+        assert (annotation.get('return') and annotation['return'] is Doc), \
+            ValueError(f"Invalid SpaCy Pipeline. Expected return type is {Doc} "
+                       f"but pipeline returns: {annotation.get('return', None)}")
+
+        self.model = model
+
+    @classmethod
+    def load_model(cls, path: str):
+        """Load and return SpaCy pipeline"""
+        try:
+            from ...nlptest import HARNESS_CONFIG as harness_config
+            
+            config = harness_config['model_parameters']
+            tgt_lang = config.get('target_language')
+        
```

Comment:
Why do we need these?

Reply:
We are specifying the target language in the model parameters in the config. And to fetch it, we are doing it (as many models support some specific target languages.)